gp2gp-translator/src/integrationTest/resources/e2e-mapping/output-json/PWTP10-output.json

```diff
@@ -2835,20 +2869,18 @@
           "extension": [ {
             "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid",
             "extension": [ {
-              "url": "descriptionId",
-              "valueId": "249995012"
```

Comment:
Valid description id, it looks like we've got a pattern here in that we're losing descriptionId's against valid description text. Investigation required.